src/lib/viewers/BaseViewer.js

```diff
@@ -53,6 +53,9 @@ const ANNOTATION_CLASSES = {
     [AnnotationMode.REGION]: CLASS_ANNOTATIONS_CREATE_REGION,
 };
 
+const IMAGE_FTUX_CURSOR_DISABLED_KEY = 'image-ftux-cursor-disabled';
```

Comment:
We should try to use positive conditions to reduce double-negatives. How about `bp-ftux-image-cursor-seen`?

Reply:
That sounds good, thanks for the suggestion.